.circleci/config.yml

```diff
@@ -182,6 +182,27 @@ jobs:
       - store_artifacts:
           path: ./reports/junit
 
+  front-detect-deadcode:
```

Comment:
`config.yml` is getting bigger and bigger :slightly_frowning_face: 
Maybe we could decrease its size using CircleCI orbs https://circleci.com/orbs/
I definitely don't insist on it, it's pretty good as it is now.

Reply:
Maybe something to bring up with Amy?
Would consider it out of scope for this PR 🙂